cmd/kops/create_cluster.go

```diff
@@ -475,7 +475,7 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 
 		// Default to a DNS name for the bastion
 		cluster.Spec.Topology.Bastion = &api.BastionSpec{
-			BastionPublicName: "bastion-" + clusterName,
+			BastionPublicName: "bastion." + c.DNSZone,
```

Comment:
I am guessing that:
```
BastionPublicName: "bastion." + c.DNSZone
```

`c.DNSZone` is not populated and needs to be used later.  We also have developer office hours where you can swing by.  Or catch one of us on slack!